src/Microsoft.AspNetCore.Html.Abstractions/HtmlTextWriter.cs

```diff
@@ -1,49 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.IO;
-
-namespace Microsoft.AspNetCore.Html
-{
-    /// <summary>
-    /// A <see cref="TextWriter"/> which supports special processing of <see cref="IHtmlContent"/>.
-    /// </summary>
-    public abstract class HtmlTextWriter : TextWriter
```

Comment:
All the places that used to use this to "flatten" will be updated to support the new Copy and Move apis.